wv2util/AppOverride.cs

```diff
@@ -777,6 +842,79 @@ public bool IsRuntimeEvergreenPreview
                 OnPropertyChanged("IsRuntimeEvergreenPreview");
             }
         }
+
+        public bool IsRuntimeStable
+        {
+            get => ReverseSearchOrder && RuntimePath == "" && ReleaseChannels.Contains("0");
+            set
+            {
+                if (value)

```

Comment:
Looks like you're missing the else case where you uncheck the box and want to remove the channel from ReleaseChannels?

It might be easier to keep the underlying data as a flags enum and have methods to go to/from the "0,1,3" style string in the serialization to env var / registry value code than have the code that knows how to edit the string to add or remove the right ",N" value